src/promptflow/promptflow/_sdk/operations/_run_operations.py

```diff
@@ -107,6 +107,24 @@ def create_or_update(self, run: Run, **kwargs) -> Run:
         except RunExistsError:
             raise RunExistsError(f"Run {run.name!r} already exists.")
 
+    @monitor_operation(activity_name="pf.runs._resume", activity_type=ActivityType.PUBLICAPI)
+    def _resume(self, resume_from: str, **kwargs) -> Run:
```

Comment:
Seems the function name should be something like "_create_by_resuming_from"?
And should it be a public function?

Reply:
I'll take the name proposed, thanks.